config/src/config/indexer_table_info_config.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+// Useful defaults
+pub const DEFAULT_PARSER_TASK_COUNT: u16 = 20;
+pub const DEFAULT_PARSER_BATCH_SIZE: u16 = 1000;
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct IndexerTableInfoConfig {
+    /// Enable table info parsing
+    pub enabled: bool,
+
+    /// Number of processor tasks to fan out
+    pub parser_task_count: u16,
+
+    /// Number of transactions each parser will process
+    pub parser_batch_size: u16,
+
+    pub enable_verbose_logging: bool,
```

Comment:
This is now called enable_expensive_logging. Sorry for the confusion we had follow up PRs.